back-end/api/rental_listing_api.py

```diff
@@ -15,4 +15,23 @@ def get(self, listing_id=None):
         
         rentals = RentalListing.query.all()
         return jsonify([rental.serialize for rental in rentals])
```

Comment:
rental.serialize(). Should be a method call not an attribute.

Reply:
thanks for this. it skipped me